src/pages/docs/tutorials/setup/index.md

```diff
@@ -25,16 +25,18 @@ Once installed, you can run `enact` from anywhere to run a command.
 
 The first command you'll use is `enact create [<directory>]` to create a new application. The `[<directory>]` argument is optional and defaults to the current working directory. `create` creates the [initial directory structure](#directory-structure) and [configures the app](#configuring-the-application).
 
+> **Note**: If you are developing an app for a webOS-based system (webOS TVs, [webOS OSE](https://www.webosose.org/), etc.) you can use the `webostv` template to create your application: `enact create -t webostv [<directory>]`
```

Comment:
Template must be installed first.  Add a link to how to install templates?

Reply:
I thought it existed in the base `cli`?  I'll check and update :+1: